temporal-testing/build.gradle

```diff
@@ -53,6 +55,23 @@ task testServiceServer(type: CreateStartScripts) {
     dependsOn 'sourcesJar'
 }
 
+graal {
+  outputName "temporal-test-server"
+  mainClass application.mainClassName
+  javaVersion '17'
+  graalVersion '21.3.0'
+  option "--class-path=${buildDir}/dependencies/*"
+  option "--allow-incomplete-classpath"
+  option "--no-fallback"
+  option "--install-exit-handlers"
```

Comment:
It would be great if we can provide some commentary for each of the options that you still remember why were added :)